bundle.Dockerfile

```diff
@@ -13,7 +13,7 @@ LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 
 LABEL com.redhat.delivery.operator.bundle=true
-LABEL com.redhat.openshift.versions="v5.1"
+LABEL com.redhat.openshift.versions="v4.7"
```

Comment:
accidentally added?

Reply:
this LABEL is the version of OCP index image in which this bundle will be added.
We added our 5.0 bundles into OCP 4.7. We could change this to 4.8 if Logging 5.0 lands on OCP 4.8 instead of 4.7